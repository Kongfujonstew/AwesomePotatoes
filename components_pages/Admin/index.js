// import PageContainer from '../PageContainer';
import Link from 'next/link';
import styled from 'styled-components';
import MoviesWithData from '../../components_apollo/MoviesWithData';
import CreateMovieWithData from '../../components_apollo/CreateMovieWithData';


const Admin = styled.div`

`


export default () => (
  <Admin id="admin">
    <CreateMovieWithData />
    <MoviesWithData />
  </Admin>
);


