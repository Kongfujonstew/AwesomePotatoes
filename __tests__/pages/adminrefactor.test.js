import React from 'react';
// import Link from '../Link.react';
import renderer from 'react-test-renderer';

import Admin from '../../pages/adminrefactor';

it('renders correctly', () => {
  const tree = renderer
    .create(<Admin/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});