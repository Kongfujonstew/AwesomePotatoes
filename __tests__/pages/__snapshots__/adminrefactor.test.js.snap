// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa cKRXsU"
  id="layout"
>
  <header>
    <nav>
      Public nav: 
      <a
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
       |
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
       |
      <a
        href="/contact"
        onClick={[Function]}
      >
        Contact
      </a>
    </nav>
    <nav>
      Admin nav: 
      <a
        href="/admin"
        onClick={[Function]}
      >
        Admin
      </a>
       |
      <a
        href="/profile"
        onClick={[Function]}
      >
        User Profile
      </a>
    </nav>
    <div>
      <div>
        Please log in.
         
        <br />
        <button
          onClick={[Function]}
        >
          Login
        </button>
      </div>
    </div>
  </header>
  <div
    className="sc-htpNat gPdQJU"
    id="admin"
  >
    <div>
      <form>
        <h3>
          ADD MOVIE
        </h3>
        <div>
          You must log in to add movies
        </div>
        <p>
          Name:
        </p>
        <input
          id="name"
          type="text"
        />
        <p>
          Description:
        </p>
        <input
          id="description"
          type="text"
        />
        <br />
        <button
          onClick={[Function]}
        >
          Add
        </button>
      </form>
    </div>
    <div
      id="cards"
    >
      Loading movie data . . .
    </div>
  </div>
  <footer>
    Jon Michael Stewart
  </footer>
</div>
`;
